torch/_dynamo/convert_frame.py

```diff
@@ -595,6 +603,10 @@ def _compile(
     output: Optional[OutputGraph] = None
     tracer: Optional[InstructionTranslator] = None
 
+    tf_mode_stack: List[
+        torch.overrides.TorchFunctionMode
+    ] = torch.overrides._get_current_function_mode_stack()
```

Comment:
```suggestion
    ] = [*torch.overrides._get_current_function_mode_stack()]
```
Defensively make copy in case someone mutates it.

Reply:
This function already makes a list copy internally (the stack itself is a C++ vector which is read)